test/functional/search_api_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+
+class SearchApiTest < ActiveSupport::TestCase
+  include Rack::Test::Methods
+
+  def app
+    Rails.application
+  end
+
+  test 'search notes functionality' do
+    get '/api/srch/notes?srchString=Blog'
+    assert last_response.ok?
+
+    # Expected search pattern
+    pattern = {
+      srchParams: {
+        srchString: 'Blog',
+        seq: false,
+      }.ignore_extra_keys!
+    }.ignore_extra_keys!
+
+    matcher = JsonExpressions::Matcher.new(pattern)
+
+    assert matcher =~ JSON.parse(last_response.body)
```

Comment:
OK, so this comparison didn't work -- we've probably missed some of the content on lines 16-18. Could be best to do `puts JSON.parse(last_response.body)` to see what comes out, and we can work from there!